prereise/gather/winddata/hrrr/helpers.py

```diff
@@ -0,0 +1,56 @@
+from datetime import timedelta
+
+
+def dt_iter(start_dt, end_dt, delta=timedelta(days=1)):
+    """Iterates from a start datetime (inclusive) to a end datetime (exclusive)
+    with step size of delta
+
+    :param datetime.datetime start_dt: datetime to start at
+    :param datetime.datetime end_dt: datetime to end at
+    :param datetime.timedelta delta: delta to step increase with
+
+    :return: (*Generator[datetime.datetime]*)
+    """
+    while start_dt < end_dt:
+        yield start_dt
+        start_dt += delta
```

Comment:
Could we use the pandas `date_range` function (perhaps with the `to_pydatetime` method of the returned `DatetimeIndex` object) instead of rolling our own?

Reply:
nice yeah, didn't know that existed